src/healthData/HealthDataNotification.js

```diff
@@ -0,0 +1,93 @@
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, Mustache, $ */
+
+define(function (require, exports, module) {
+    "use strict";
+    
+    var AppInit                      = require("utils/AppInit"),
+        PreferencesManager           = require("preferences/PreferencesManager"),
+        Strings                      = require("strings"),
+        Dialogs                      = require("widgets/Dialogs"),
+        HealthDataNotificationDialog = require("text!htmlContent/healthdata-notification-dialog.html"),
+        HealthDataPreview            = require("healthData/HealthDataPreview"),
+        UrlParams                    = require("utils/UrlParams").UrlParams;
+    
+    PreferencesManager.definePreference("healthDataNotification", "number", 0);
+    
+    var prefs = PreferencesManager.getExtensionPrefs("healthData");
+    
+    // Parse URL params
+    var params = new UrlParams();
+    /**
+     * Show dialog for fist time to the user regarding log capturing by Brackets
+     */
+    function showDialogHealthDataNotification() {
+        var hdPref   = prefs.get("healthDataTracking"),
+            template = Mustache.render(HealthDataNotificationDialog, {"Strings": Strings, "hdPref": hdPref}),
+            $template = $(template),
+            newHDPref = hdPref;
+
+        $template.on("change", "[data-target]:checkbox", function () {
+            newHDPref = $(this).is(":checked");
+        });
+
+        Dialogs.showModalDialogUsingTemplate($template).done(function (id) {
+            PreferencesManager.setViewState("healthDataNotification", (new Date()).getTime());
+     
+            if (id === "save") {
+                if (hdPref !== newHDPref) {
+                    prefs.set("healthDataTracking", newHDPref);
+                }
+            }
+        });
+    }
+
+    function handleHealthDataStatistics() {
+        var hdPref = prefs.get("healthDataTracking");
+        
+        if (hdPref) {
+            HealthDataPreview.previewHealthDataFile();
+        } else {
+            showDialogHealthDataNotification();
+        }
```

Comment:
I wonder if we could just do `HealthDataPreview.previewHealthDataFile()` either way.  It seems like the preview dialog is basically a superset of the notification dialog.  And if we were to streamline the notification more (e.g. a message bar like what Firefox uses), then it really couldn't be used here...